libnavigator/src/main/java/com/mapbox/navigation/navigator/LocationEx.kt

```diff
@@ -38,7 +38,7 @@ internal fun Location.toFixLocation(date: Date): FixLocation {
     return FixLocation(
         Point.fromLngLat(longitude, latitude),
         elapsedRealtimeNanos,
-        date,
+        Date(time),
```

Comment:
The idea behind this was to exact mirror Android's location cc @SiarheiFedartsou